File: src/main/java/kitchenpos/product/tobe/infra/JpaProductRepository.java
package kitchenpos.product.tobe.infra;

import kitchenpos.product.tobe.domain.Product;
import kitchenpos.product.tobe.domain.ProductRepository;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import java.util.UUID;

@Repository("newProductRepository")
public interface JpaProductRepository extends ProductRepository, JpaRepository<Product, UUID> {
}

File: src/main/kotlin/kitchenpos/product/tobe/infra/JpaProductRepository.kt
package kitchenpos.product.tobe.infra

import kitchenpos.product.tobe.domain.Product
import kitchenpos.product.tobe.domain.ProductRepository
import org.springframework.data.jpa.repository.JpaRepository
import java.util.*

interface JpaProductRepository : ProductRepository, JpaRepository<Product, UUID>
